libs/accounts/src/lib/accounts-manager.spec.tsx

```diff
@@ -66,7 +66,7 @@ describe('AccountManager', () => {
 
     it('update method should return proper result', async () => {
       let rerenderer: (ui: React.ReactElement) => void;
-      await act(() => {
+      await act(async () => {
```

Comment:
non-blocking: don't think `async` here is necessary: return value of act is always a thenable, so `await` is useful, but `async` on callback if there is no any `await` inside a callback is redundant